barretenberg/cpp/src/barretenberg/vm/avm/generated/prover.cpp

```diff
@@ -28,18 +28,9 @@ using FF = Flavor::FF;
  */
 AvmProver::AvmProver(std::shared_ptr<Flavor::ProvingKey> input_key, std::shared_ptr<PCSCommitmentKey> commitment_key)
     : key(std::move(input_key))
+    , prover_polynomials(*key)
```

Comment:
This is probably the right thing but can you confirm there are no performance regression?